activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -227,6 +227,31 @@ def last(*args)
         @association.last(*args)
       end
 
+      # Gives a record (or N records if a parameter is supplied) from the collection
+      # Uses the same rules as <tt>ActiveRecord::Base.take</tt>.
```

Comment:
Thanks for the PR! :smile:

Is this supposed to be two sentences or one? If it's 2 can you add the period at the end of the first sentence? If it's 1 then lowercase uses and change it to using. Thanks!